<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDialog</class>
 <widget class="QDialog" name="EventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evento de Calendario</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Título:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="title_edit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="time_layout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hora inicio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="start_time">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hora fin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="end_time">
       <property name="displayFormat">
        <string>HH:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Descripción:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="description_edit">
     <property name="maximumHeight">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>